src/test/groovy/me/champeau/gradle/ClasspathIsUsedFunctionalTest.groovy

```diff
@@ -30,7 +30,7 @@ class ClasspathIsUsedFunctionalTest extends BaseFunctionalTest {
                 throw new IllegalStateException("Need to update the byteCode version mapping for Java ${JavaVersion.current()}, you can ref https://javaalmanac.io/bytecode/versions")
         }
         // Superclasses can only be reported if the classpath is present
-        hasTextReport("""
+        hasTextReport("""Comparing source compatibility of ${testProjectDir.fileName}-v2.jar against ${testProjectDir.fileName}.jar
```

Comment:
```suggestion
        hasTextReport("""
        Comparing source compatibility of ${testProjectDir.fileName}-v2.jar against ${testProjectDir.fileName}.jar
```

Reply:
@Goooler There's no `\n` before `Comparing source compatibility` so this change makes the test fail.

Reply:
Ooops, and need a `.trim()` in the end.

Reply:
If the problem is that this statement is not very readable, we could split it into two different `hasTextReport` checks.

```suggestion
        hasTextReport("Comparing source compatibility of ${testProjectDir.fileName}-v2.jar against ${testProjectDir.fileName}.jar")
        hasTextReport("""
`````````